### PR TITLE
Feature/5189

### PR DESCRIPTION
**Pull request by jay-lee00**

### CODE REVIEW

File: SnapMD.VirtualCare.ApiModels/ScheduledConsultationResult.cs [hunk 1]
```diff
@@ -13,5 +14,10 @@ public class ScheduledConsultationResult
         /// The ID of the newly-created consultation.
         /// </summary>
         public int ConsultationId { get; set; }
+
+        /// <summary>
+        /// The UID of the newly-created consultation request.
```
Comment (lorddev) [inline SnapMD.VirtualCare.ApiModels/ScheduledConsultationResult.cs]: @jay-lee00 What is meant by UID?

Comment (jay-lee00) [inline SnapMD.VirtualCare.ApiModels/ScheduledConsultationResult.cs]: @lorddev  The idea is to distinguish the GUID based unique ID from the INT based ID.

Comment (lorddev) [inline SnapMD.VirtualCare.ApiModels/ScheduledConsultationResult.cs]: @jay-lee00 Integer IDs are expected to be legacy usage. As of v2.1 API many IDs are GUIDs, and our documentation ought to be enough for clients to understand this. For example if we see "ConsultationId" we know it's going to be an int. If we see "AppointmentId", "PersonId", "ParticipantId", we know it's going to be a Guid.

**Comment by jay-lee00** [comment]
@lorddev updated.

**Comment by lorddev** [comment]
@jay-lee00 Can you check the merge conflicts please?

**Comment by jay-lee00** [comment]
this is old, already included in 1.24.

**Pull request closed without merging**